docs/source/dev_guide/release.rst

```diff
@@ -3,176 +3,83 @@ How to Prepare a Release
+        $ git add --update
+        $ git commit --message Bump to 1.2.0
+        $ git push origin v1.2.0
+        :: Looking good? (y/N)
+        >
+4. Create a pull request to the main repo and merge it.
 
 .. _github-release:
 
-Creating A Release On GitHub
-----------------------------
-
-1. Go to the Releases on the GitHub repo of the project
-`here <https://github.com/E3SM-Project/zstash/releases>`_.
-and draft a new release.
-
-2. ``Tag version`` and ``Release title`` should both be the version, including the "v".
-(They should match ``git_rev`` in step 4 of the previous section).
-``Target`` should be ``master``. Use ``Describe this release`` to write what features
-the release adds. You can scroll through
-`Zstash commits <https://github.com/E3SM-Project/zstash/commits/master>`_ to see
-what features have been added recently.
-
-Note that you can also change the branch which you want to release from,
-this is specified after the tag (@ Target: ``master``).
+Releasing on GitHub
+-------------------
 
-The title of a release is often the same as the tag, but you can set it to whatever you want.
+1. Draft a new release `here <https://github.com/E3SM-Project/e3sm_diags/releases>`_.
+2. Set `Tag version` to ``v<version>``, **including the "v"**. `@Target` should be ``master``.
+3. Set `Release title` to ``v<version>``, **including the "v"**.
+4. Use `Describe this release` to summarize the changelog.
 
-Remember to write a description.
+   * You can scroll through `zstash commits <https://github.com/E3SM-Project/zstash/commits/main>`_ for a list of changes.
 
-.. figure:: github_release.png
-    :figwidth: 100 %
-    :align: center
-    :target: github_release.png
+5. If this version is a release candidate (``<version>`` appended with ``rc``), checkmark `This is a pre-release`.
+6. Click `Publish release`.
+7. CI/CD release workflow is automatically triggered.
 
-    An example of a completed page to release the code
-
-3. Click "Publish release".
-
-Updating The sha256
---------------------
-
-1. Download "Source code (.tar.gz)" from the `GitHub releases page <https://github.com/E3SM-Project/zstash/releases>`_.
-
-2. Run ``shasum -a 256`` on this file. For example:
-
-    ::
-
-        shasum -a 256 zstash-0.4.1.tar.gz
-
-3. On your machine, pull the latest version of the code.
-This will have the ``conda/meta.yaml`` we edited in the first section.
-
-    ::
-
-        git checkout master
-        git pull <upstream-origin> master
-
-Or:
-    ::
-
-        git fetch <upstream-origin> master
-        git checkout -b <branch-name> <upstream-origin>/master
-
-4. Change ``sha256`` in ``conda/meta.yaml`` to the result of step 2.
-
-5. Commit and push your changes.
-
-    ::
-
-        git commit -am 'Edit sha256 for v0.4.1'
-        git push <upstream-origin> master
-
-Or:
-
-    ::
-
-        git commit -am 'Edit sha256 for v0.4.1'
-        git push <fork-origin> <branch-name>
-        # Create pull request for the master branch
```

Comment:
No longer necessary since Anaconda packages are released in CI/CD using source from git rev, rather than the SHA.